csharp/Theorafile.cs

```diff
@@ -237,9 +240,9 @@ public static int tf_close(ref IntPtr file)
 	private static IntPtr AllocTheoraFile()
 	{
 		// Do not attempt to understand these numbers at all costs!
-		const int size32 = 1160;
-		const int size64Unix = 1472;
-		const int size64Windows = 1328;
+		const int size32 = 1164;
+		const int size64Unix = 1480;
+		const int size64Windows = 1336;
```

Comment:
So this part can actually be dropped - I completely forgot how large the Xiph structs can be; this change actually reduces the struct size by nearly half! So let's just keep these as they are for .NET ABI compatibility, in case someone uses an older native version.

Reply:
Removed.